packages/grid/_modules_/grid/models/gridFilterItem.ts

```diff
@@ -3,6 +3,7 @@ export interface GridFilterItem {
   columnField?: string;
   value?: string;
   operatorValue?: string;
+  operatorLabel?: string;
```

Comment:
we already have a label in `GridFilterOperator`, can we use this one?

Reply:
that is the object that is being passed to the tooltip. Basically, if the filter operator has a filter it will pass it along. Since the numeric operators are the only ones that have a label only then will that value be present? It doesn't add anything new, only passes values along. and it is optional.

Reply:
I see.
The issue is that now we are exposing this new prop to the users as part of the filter model. Also it will only work in the toolbar button as we use the operator label in the filter form.
So I think, we need to load the operators from the `ColDef` in the toolbar button, so we get its label...